targets/TARGET_STMBLUE/Periph_Driver/inc/BlueNRG1_rtc.h

```diff
@@ -0,0 +1,242 @@
+/**
+  ******************************************************************************
+  * @file    BlueNRG1_rtc.h
+  * @author  VMA Application Team
+  * @version V2.0.0
+  * @date    21-March-2016
+  * @brief   This file contains all the functions prototypes for the RTC 
+  *          firmware library.
+  ******************************************************************************
+  * @attention
+  *
+  * THE PRESENT FIRMWARE WHICH IS FOR GUIDANCE ONLY AIMS AT PROVIDING CUSTOMERS
+  * WITH CODING INFORMATION REGARDING THEIR PRODUCTS IN ORDER FOR THEM TO SAVE
```

Comment:
Also this Periph driver - needs license header here

Reply:
Hi @0xc0170, on the next commit I'll provide a link to ST webpage where the license is located inside the header of each file. In addition to that I'll include a PDF file with license in the root target directory.

Reply:
License should follow our guidance, instead of pdf having it as "LICENSE-.....txt as for other licenses in this. What license is it btw?
Each file should contain SPDX identifier and license header. 

Will wait for an update to review.

Reply:
I'm waiting a reply from ST legal department about the distribution of the license file in *.txt file instead of the original PDF file. It is located at
https://www.st.com/content/ccc/resource/legal/legal_agreement/license_agreement/group0/12/e6/90/c9/41/55/40/1d/DM00279991/files/DM00279991.pdf/jcr:content/translations/en.DM00279991.pdf
I asked also to provide me a complete header including in some way this information.